occurrence-es-mapping/src/main/java/org/gbif/occurrence/search/es/SearchHitConverter.java

```diff
@@ -113,6 +113,30 @@ public abstract class SearchHitConverter<T> implements Function<SearchHit, T> {
 
   protected final OccurrenceBaseEsFieldMapper occurrenceBaseEsFieldMapper;
 
+  protected Optional<String> getStringValue(SearchHit hit, ChecklistEsField esField, String defaultChecklistKey) {
```

Comment:
maybe rename these 2 methods to sth like getChecklistStringValue or sth along those lines? just to make clearer the difference between with the overloaded methods

Reply:
done